sshd.c

```diff
@@ -884,16 +897,16 @@ privsep_postauth(Authctxt *authctxt)
 		    posix_spawn_file_actions_adddup2(&actions, tmp_sock, STDIN_FILENO) != 0 ||
 		    posix_spawn_file_actions_adddup2(&actions, tmp_sock, STDOUT_FILENO) != 0 ||
 		    posix_spawn_file_actions_adddup2(&actions, pmonitor->m_recvfd, PRIVSEP_MONITOR_FD) != 0)
-			error("posix_spawn initialization failed");
-		else {
+			fatal("posix_spawn initialization failed");
+		
+		{
```

Comment:
remove the braces {, }

Reply:
they need to be there, since there is a variable declaration right below. Conforming to C standard, all variable declarations should be the at the top of code block.

Reply:
Not to be pragmatic, but any compiler in the last decade is probably compliant with C99 which allows you to declare variables anywhere (i.e. use mixed declarations).

Reply:
Agree.. but there is no harm in being in sync with rest of the code and be as portable as possible.